controller/api/public/client.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	"github.com/golang/protobuf/proto"
 	common "github.com/runconduit/conduit/controller/gen/common"
+	healthcheckPb "github.com/runconduit/conduit/controller/gen/common/healthcheck"
```

Comment:
TIOLI but I'd like to consistently use the same prefix every time we import these protos

Reply:
I'm on the fence about this naming convention. I agree that it makes sense in a file where we import multiple protobuf definitions, but in a file like `healthcheck.go`, which only imports the healthcheck protobuf, it feels more concise to just call it `pb`.

Reply:
Actually, on second thought I think you're right that this would be clearer with the same import name throughout. Will update in a subsequent commit.